app/src/main/java/com/antailbaxt3r/collegemate/activities/LoginActivity.java

```diff
@@ -124,16 +128,20 @@ private void handleSignInResult(GoogleSignInResult result) {
         }
     }
 
-    private void firebaseAuthWithGoogle(AuthCredential credential) {
+    private void firebaseAuthWithGoogle(String idToken) {
+        AuthCredential credential = GoogleAuthProvider.getCredential(idToken, null);
```

Comment:
Don't change this,

Reply:
Needed this to implement OAuth verification function.